src/coreclr/pal/src/arch/riscv64/dispatchexceptionwrapper.S

```diff
@@ -1,13 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-
-//
-// Implementation of the PAL_DispatchExceptionWrapper that is
-// interposed between a function that caused a hardware fault
-// and PAL_DispatchException that throws an SEH exception for
-// the fault, to make the stack unwindable.
-//
-
-#include "unixasmmacros.inc"
-
-#error "TODO-RISCV64: missing implementation"
```

Comment:
It seems that this change should be reverted with `#error` line commented out like in walker.cpp above

Reply:
Okay. I reverted and commented out `#error "TODO-RISCV64: missing implementation"`